### DISCUSSION TITLE
right formatting of Array init

### DISCUSSION THREAD

**Discussion by ingDIY** [uncrustify/uncrustify#4329] [Q&A]
Hello!
I would like to format an array init like this:
const char firmware_ver[] = {
		( digit10_1( FW_MAJOR ) + '0' ), 
		( digit10_0( FW_MAJOR ) + '0' ), '.',
		( digit10_1( FW_MINOR ) + '0' ),
		( digit10_0( FW_MINOR ) + '0' ), '.',
		( digit10_1( FW_PATCH ) + '0' ),
		( digit10_0( FW_PATCH ) + '0' ), '\0'};

I tried several settings but I uncrustify give me a crazy results with strange tabs:
![Immagine 2024-06-20 163556](https://github.com/uncrustify/uncrustify/assets/10012263/0feefb8a-80bf-400f-8929-3ed922f64ba5)

What options should I check?
Thanks!

PS: i attach my current config file
[uncrustify_config.txt](https://github.com/user-attachments/files/15915760/uncrustify_config.txt)

**Comment by guy-maurel** [comment]
please, create a new Issue, provide your input file, the config and the expected output file
